apps/desktop/src/app/layout.tsx

```diff
@@ -1,26 +1,40 @@
-import type { Metadata } from "next";
+"use client";
+
 import { Inter } from "next/font/google";
-import "./globals.css";
+import { createContext, useState } from "react";
+
 import { Navbar } from "../components/navbar";
+import "./globals.css";
 
 const inter = Inter({ subsets: ["latin"] });
 
-export const metadata: Metadata = {
-	title: "Create Next App",
-	description: "Generated by create next app",
-};
+interface ThemeState {
+	lightMode: boolean;
+	setLightMode: (value: boolean) => void;
+}
+
+export const ThemeContext = createContext<ThemeState>({
+	lightMode: false,
+	setLightMode: (value: boolean) => {
+		throw new Error("Context used without provider");
+	},
+});
 
 export default function RootLayout({
 	children,
 }: Readonly<{
 	children: React.ReactNode;
 }>) {
+	const [lightMode, setLightMode] = useState(false);
+
 	return (
-		<html lang="en">
-			<body className={inter.className}>
-				<Navbar />
-				{children}
-			</body>
-		</html>
+		<ThemeContext.Provider value={{ lightMode, setLightMode }}>
+			<html lang="en">
+				<body className={inter.className}>
+					<Navbar />
+					{children}
+				</body>
+			</html>
+		</ThemeContext.Provider>
```

Comment:
We need to move the context provider into the root layout, otherwise it only exists for `/`